### PR TITLE
Docs updates (mainly removing debug_mode)

### LINKED ISSUE #310: Docs mention debug_mode but it has been replaced with Verbosity
### Description

The current docs mention debug_mode but it is never used. Verbosity is used instead. Need to change docs to reflect that

### Example

https://openmdao.github.io/Aviary/user_guide/outputs_and_how_to_read_them.html

### Aviary Version

0.9.4-dev

### Relevant environment information

_No response_

### PR DESCRIPTION
**Pull request by xjjiang**
### Summary

- `debug_mode` is replaced by `Settings.VERBOSITY` in docs and visualization.
- added an explanation of tilde (~) in `variable_meta_data.py`

### Related Issues

- Resolves #310 

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/docs/getting_started/onboarding_level1.ipynb [hunk 1]
```diff
@@ -691,7 +687,7 @@
    "source": [
     "### Plain text inputs and outputs\n",
     "\n",
-    "If `debug_mode` is set to `True`, `input_list.txt` and `output_list.txt` are generated. We recommend this parameter be set to `True` for beginners. \n",
+    "If `Settings.VERBOSITY` is set to `2` or higher, `input_list.txt` and `output_list.txt` are generated. We recommend this parameter be set to `True` for beginners. \n",
```
Comment (Kenneth-T-Moore) [inline aviary/docs/getting_started/onboarding_level1.ipynb:690]: The last part of this sentence "recommend this parameter be set to 'True'" is no longer the case.  I don't know what verbosity we should recommend for beginners, but maybe not the highest.

Comment (crecine) [reply aviary/docs/getting_started/onboarding_level1.ipynb:690]: I believe brief is the default for everything, except tests which are Quiet

**Comment by crecine** [comment]
verbosity is intended to be set through the command line, though I believe it is still possible to set it through the CSV

File: aviary/docs/getting_started/onboarding_level1.ipynb (hunk 1 above)
Comment (xjjiang) [reply aviary/docs/getting_started/onboarding_level1.ipynb:690]: Modified as:

If `Settings.VERBOSITY` is set to `2` (Verbosity.VERBOSE) or higher, `input_list.txt` and `output_list.txt` are generated. We recommend this parameter be set to `1` (Verbosity.BRIEF) for beginners.

**Review by Kenneth-T-Moore — APPROVED** [review]

**Review by jkirk5 — APPROVED** [review]

**Pull request merged**